package.json

```diff
@@ -2,53 +2,55 @@
   "name": "ember-singularity",
   "version": "1.0.2",
   "description": "Unified event handling for Ember.",
+  "keywords": [
+    "ember-addon",
+    "event",
+    "events"
+  ],
+  "license": "MIT",
+  "author": "Trent Willis <trentmwillis@gmail.com>",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
-  "scripts": {
-    "start": "ember server",
-    "build": "ember build",
-    "test": "ember try:each"
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/trentmwillis/ember-singularity"
   },
-  "engines": {
-    "node": ">= 0.10.0"
+  "scripts": {
+    "build": "ember build",
+    "start": "ember server",
+    "test": "ember try:each"
+  },
+  "dependencies": {
+    "ember": "^1.0.3",
+    "ember-cli-babel": "^6.6.0",
+    "ember-source": "^2.16.2",
```

Comment:
I believe this should just be a `devDependency`, as we don't want end users to potentially install two versions of Ember.